pom.xml

```diff
@@ -432,7 +432,7 @@
         <version>${commons-lang3.version}</version>
       </dependency>
       <dependency>
-        <groupId>org.apache.commons</groupId>
```

Comment:
> This link says that common-lang was moved from common-lang group to org.apache.commons group. 
> http://mvnrepository.com/artifact/commons-lang/commons-lang

Right. But it's under `commons-lang3`: http://mvnrepository.com/artifact/org.apache.commons/commons-lang3 since they cannot change the old artifact.